MODULE_ACCEPTANCE_CRITERIA_TEMPLATE.MD

```diff
@@ -0,0 +1,49 @@
+# Module acceptance criteria template
+
+## Conventions
```

Comment:
Maybe expand this or add another section explaining "how to use this form".  Make it explicitly clear to those using this.